src/transformers/models/cvt/configuration_cvt.py

```diff
@@ -0,0 +1,159 @@
+        mlp_ratios (`List[float]`, *optional*, defaults to [4.0, 4.0, 4.0, 4.0]):
+            Ratio of the size of the hidden layer compared to the size of the input layer of the Mix FFNs in the
+            encoder blocks.
+        attention_drop_rate (`List[float]`, *optional*, defaults to [0.0, 0.0, 0.0]):
+            The dropout ratio for the attention probabilities.
+        drop_rate (`List[float]`, *optional*, defaults to [0.0, 0.0, 0.0]):
+            The dropout ratio for the patch embeddings probabilities.
+        drop_path_rate (`List[float]`, *optional*, defaults to [0.0, 0.0, 0.1]]):
+            The dropout probability for stochastic depth, used in the blocks of the Transformer encoder.
+        qkv_bias (`List[bool]`, *optional*, defaults to [True, True, True]]):
+            The bias bool for query, key and value in attentions
+        cls_token (`List[bool]`, *optional*, defaults to [False, False, True]]):
+            The bool for classification token
+        pos_embed (`List[bool]`, *optional*, defaults to [False, False, True]]):
+            The bool for position embeddings
+        cls_token (`List[bool]`, *optional*, defaults to [False, False, True]]):
+            The bool for cls_token
+        qkv_projection_method (`List[string]`, *optional*, defaults to 'dw_bn', 'dw_bn', 'dw_bn']]):
+            The projection method for query, key and value Default is depth-wise convolutions with batch norm. For
+            Linear projection use "avg".
+        kernel_qkv (`List[int]`, *optional*, defaults to [3, 3, 3]):
+            The kernel size for query, key and value in attention layer
+        padding_kv (`List[int]`, *optional*, defaults to [1, 1, 1]):
+            The padding size for key and value in attention layer
+        stride_kv (`List[int]`, *optional*, defaults to [2, 2, 2]):
+            The stride size for key and value in attention layer
+        padding_q (`List[int]`, *optional*, defaults to [1, 1, 1]):
+            The padding size for query in attention layer
+        stride_q (`List[int]`, *optional*, defaults to [1, 1, 1]):
+            The stride size for query in attention layer
+        initializer_range (`float`, *optional*, defaults to 0.02):
+            The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
+        layer_norm_eps (`float`, *optional*, defaults to 1e-6):
+            The epsilon used by the layer normalization layers.
+        Example:
+
+    ```python
+    >>> from transformers import CvtModel, CvtConfig
+
+    >>> # Initializing a Cvt msft/cvt style configuration
+    >>> configuration = CvtConfig()
+
+    >>> # Initializing a model from the msft/cvt style configuration
+    >>> model = CvtModel(configuration)
+
+    >>> # Accessing the model configuration
+    >>> configuration = model.config
+    ```"""
+    model_type = "cvt"
+
+    def __init__(
+        self,
+        image_size=224,
```

Comment:
To be removed

Reply:
Needed for image size during conversion of weight scripts